emmanuel-mukira/Group-2-task-management-app
# Feat: Labels component

Description
Changed the folder structure to use the root files for HTML & CSS.
Ticket [LABELS AND LABEL DETAILS](https://app.clickup.com/t/866a67gaz)
Types Of Changes
[] Bug fix
[] New Feature
[] Breaking Changes
Checklist:
[x]My code follows the code style of this project
[x]All new and existing tests passed.
[x]My code has been linted and formatted correctly.
[x]I have commented my code, particularly in hard-to-understand areas.
[x]My changes generate no new warnings.
How can this code be tested?
After Cloning this repo , run git pull  to ensure you are up to date.
Run the changes in the browser
